src/theme/icon.rs

```diff
@@ -62,7 +62,7 @@ impl ByType {
             file: "\u{1f4c4}".into(),
             pipe: "\u{1f4e9}".into(),
             socket: "\u{1f4ec}".into(),
-            executable: "\u{1f3d7}".into(),
```

Comment:
Let's stick to the existing one for this.

Reply:
> Let's stick to the existing one for this.

weird, it was normal before